provider/lib/utils.js

```diff
@@ -98,6 +98,9 @@ module.exports = function(
         var method = 'postTrigger';
 
         return new Promise(function(resolve, reject) {
+            var triggerIdentifier = utils.getTriggerIdentifier(dataTrigger.apikey, dataTrigger.namespace, dataTrigger.name);
```

Comment:
Is it possible for `utils.getTriggerIdentifier()` to throw an exception? If so, then the semantics have been changed.